# i2c_data.msg
float64 x
float64 y
float64 z 
float64 timestamp